.github/workflows/build-master.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: '1'
+    steps:
+      - name: print docker info
+        run: docker version && docker info
+      - uses: actions/checkout@v2
+      - name: build image
+        run: docker build --build-arg=ENABLE_ARCHIVES=true --target=current -t documentation:latest .
```

Comment:
This should probably be `--target=deploy-source` ?

Reply:
Good idea. I've updated the `docker build` as shown in the `Dockerfile` .